Tests/TensorFlowTests/OperatorTests/MathTests.swift

```diff
@@ -250,6 +250,13 @@ final class MathOperatorTests: XCTestCase {
         assertEqual(y, expectedY, accuracy: 1e-5)
     }
 
+    func testSwish() {
```

Comment:
Looks good! Could you test the gradient as well? Thanks!

Reply:
I have another PR testing gradients for a lot of these functions, I'll have that up in a few days.

Reply:
Once we have these tests, they'll verify that the compiler indeed does compute a correct vjp for the function.